pip/commands/completion.py

```diff
@@ -37,7 +38,7 @@
             set -lx PIP_AUTO_COMPLETE 1
             string split \\  -- (eval $COMP_WORDS[1])
         end
-        complete -fa "(__fish_complete_pip)" -c pip
+        complete -fa "(__fish_complete_pip)" -c %(pip_entry_point)s
```

Comment:
Could you put 2 `%%` symbols here so that you can club together the formatting into one?

Basically, the delta at 84-86 would be reset and the base completion formatting would get a new key; meaning that the formatting of the arguments then only happens in one location. :)